Cargo.lock

```diff
@@ -6635,6 +6635,7 @@ dependencies = [
 name = "gear-wasmer-cache"
 version = "1.6.2"
 dependencies = [
+ "fs2",
```

Comment:
mb switch to fork of fs2: https://github.com/al8n/fs4-rs? since it is not maintained for 7 years